help/develop/how-to/test-packages.md

```diff
@@ -174,20 +175,30 @@ Progress bar on the bottom of the page shows the percentage of completed tests.
 
 ### Reviewing results
 
-Information about test results is available via tooltip or in the context panel. Selected test, category, or package to
+Information about test results is available via tooltip or in the context panel. Select test, category, or package to
 explore results. In case category/package contain multiple tests results are shown as a grid which can be added to
 workspace for further exploration.
 
 ![Test results](test_results.gif)
 
 ## Running tests in the platform console
 
-It is possible to run tests in the platform's console. Press the tilde key `~` to open the console or enable it from the
-toolbox (`Windows | Console`). To
-launch tests for a category, type `PackageName:test(category="category-name")`,
-e.g., `ApiTests:test(category="Layouts")`, or add a specific test as a
-parameter: `PackageName:test(category="category-name", test="test-name")`, e.g.,
-`ApiTests:test(category="Layouts", test="ViewLayout.toJson()")`.
+1. **Access the Console**: To open the console, press the tilde key `~`. Alternatively, enable the console from the toolbox by navigating to `Windows` and selecting `Console`.
+
+2. **Run Tests by Category**:
+    To launch tests for a specific category, use the syntax:
+    
+    `PackageName:test(category="category-name")`
+
+    For instance, you can run tests for the "Layouts" category with: `ApiTests:test(category="Layouts")`.
+
+3. **Run Specific Tests**:
+    If you want to run a specific test within a category, include the `test` parameter.
+    The format is:
+    
+    `PackageName:test(category="category-name", test="test-name")`
+
+    For example, you can execute the test "ViewLayout.toJson()" within the "Layouts" category with: `ApiTests:test(category="Layouts", test="ViewLayout.toJson()")`.
```

Comment:
Maybe replace sentence 

> For example, you can execute the test "ViewLayout.toJson()" within the "Layouts" category with:

 with just 

> Example of usage:

Because the function signature is pretty clear. But it's controversial:)